### PR TITLE
go to tableView when add content

### PR DESCRIPTION
**Pull request by itkg-rboessel**

### CODE REVIEW

**Comment by itkg-rboessel** [comment]
@itkg-nanne

**Comment by itkg-rboessel** [comment]
@itkg-nanne @nicolasThal @itkg-ngilain @itkg-alavieille

**Comment by itkg-alavieille** [comment]
What is happening for the entity who uses the ``submitAdmin``  and not uses the dataTable

**Comment by itkg-rboessel** [comment]
$(".formCreation").data('type') != "creation" in this case so nothing change change for thme. But they don't have list.

**Comment by itkg-alavieille** [comment]
so i think you should change the parameter by something more explicit like ```data_table_creation```

**Comment by itkg-ngilain** [comment]
:+1:

File: BackofficeBundle/Resources/public/coffee/extendView/submitAdmin.coffee [hunk 1]
```diff
@@ -20,10 +20,18 @@ extendView['submitAdmin'] = {
...
-                html: response
-                submitted: true
-              ))
+              if($(".formCreation").data('type') != "listableCreation")
+                  new viewClass(viewContext.addOption(
+                    html: response
+                    submitted: true
+                  ))
+              else
+                  successMessage = $( response )[0].innerHTML
```
Comment (itkg-nanne) [inline BackofficeBundle/Resources/public/coffee/extendView/submitAdmin.coffee]: $(response).html() must do the job.

Comment (itkg-rboessel) [inline BackofficeBundle/Resources/public/coffee/extendView/submitAdmin.coffee]: @itkg-nanne that's not working. 
But i create a note to improve that by return only success message without form on next sprint.

**Pull request merged**